docs/cli.md

```diff
@@ -136,3 +143,64 @@ agent.to_cli_sync(message_history=message_history)
 ```
 
 The CLI will start with the provided conversation history, allowing the agent to refer back to previous exchanges and maintain context throughout the session.
+
+## Web Chat UI
+
+Launch a web-based chat interface by running:
+
+```bash
+clai web -m openai:gpt-5
+```
+
+This will start a web server (default: http://127.0.0.1:7932) with a chat interface.
+
+You can also serve an existing agent. For example, if you have an agent defined in `my_agent.py`:
+
+```python
+from pydantic_ai import Agent
+
+my_agent = Agent('openai:gpt-5', instructions='You are a helpful assistant.')
+```
+
+Launch the web UI:
+
+```bash
+# With a custom agent
+clai web --agent my_module:my_agent
+
+# With specific models (first is default when no --agent)
+clai web -m openai:gpt-5 -m anthropic:claude-sonnet-4-5
+
+# With builtin tools
+clai web -m openai:gpt-5 -t web_search -t code_execution
+
+# Generic agent with system instructions
+clai web -m openai:gpt-5 -i 'You are a helpful coding assistant'
+
+# Custom agent with extra instructions for each run
+clai web --agent my_module:my_agent -i 'Always respond in Spanish'
+```
+
+### Web UI Options
+
+| Option | Description |
+|--------|-------------|
+| `--agent`, `-a` | Agent to serve in [`module:variable` format](#custom-agents) |
+| `--model`, `-m` | Models to list as options in the UI (repeatable) |
+| `--tool`, `-t` | [Builtin tool](builtin-tools.md)s to list as options in the UI (repeatable). See [available tools](web.md#builtin-tool-support). |
+| `--instructions`, `-i`: System instructions. When `--agent` is specified, these are additional to the agent's existing instructions. |
+| `--host` | Host to bind server (default: 127.0.0.1) |
+| `--port` | Port to bind server (default: 7932) |
+
+When using `--agent`, the agent's configured model becomes the default. CLI models (`-m`) are additional options. Without `--agent`, the first `-m` model is the default.
+
+!!! note "Memory Tool"
+    The [`memory`](builtin-tools.md#memory-tool) builtin tool cannot be enabled via `-t memory`. If your agent needs memory, configure the [`MemoryTool`][pydantic_ai.builtin_tools.MemoryTool] directly on the agent and provide it via `--agent`.
+
+The web chat UI can also be launched programmatically using [`Agent.to_web()`][pydantic_ai.Agent.to_web], see the [Web UI documentation](web.md).
```

Comment:
This doesn't make sense under an "Options" setting; we can move this to the section above after the example